content/features/featuresDeepDive/importers.md

```diff
@@ -12,8 +12,8 @@ video-content:
 
 The built in file type is `.babylon` and Babylon.js can load these without a plugin. All other file types require a plugin as described in this section.
 
-Possible file types are gLTF, obj, stl
+Possible file types are gLTF, splat, obj, stl.
 
 To help you with imported assets there is a manager for them.
 
-**Note:** Since meshes you import can have a _rotationQuaternion_ set before applying a rotation set the _rotationQuaternion_ to _null
\ No newline at end of file
+**Note:** Since meshes you import can have a _rotationQuaternion_ set before applying a rotation set the _rotationQuaternion_ to _null.
```

Comment:
you can use the ` symbol for code parts:

```suggestion
**Note:** Since meshes you import can have a `rotationQuaternion` set before applying a rotation set the `rotationQuaternion` to `null`.
```